.pre-commit-config.yaml

```diff
@@ -10,9 +10,26 @@ repos:
       - id: detect-aws-credentials
         args: ['--allow-missing-credentials']
   - repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.53.0
+    rev: v1.62.0
```

Comment:
For now we should avoid going above v1.62.0 until https://github.com/antonbabenko/pre-commit-terraform/issues/337 is fixed without work arounds

Reply:
Fixed in v1.64.1 https://github.com/antonbabenko/pre-commit-terraform/pull/339